src/tv2-common/actions/executeAction.ts

```diff
@@ -325,7 +328,7 @@ async function getExistingTransition<
 	const transition = existingTransition.piece.name
 
 	// Case sensitive! Blueprints will set these names correctly.
-	const transitionProps = transition.match(/CUT|MIX (\d+)|EFFEKT (\d+)/)
+	const transitionProps = transition.match(/CUT|MIX (\d+)|DIP (\d+)|EFFEKT (\d+)/)
```

Comment:
Just a suggestion. The regular expression could be refactored to be case insensitive and allow for zero or more spaces instead of requiring a single space – giving a bit of flexibility. Another thing good be to group the transition types and use named groups., something like:
```suggestion
	const transitionProps = transition.match(/CUT|(?:MIX|DIP|EFFEKT) *(?<frames>\d+)/i)?.groups
```
It would make the the code more readable if the condition for MIX below was something like
```typescript
if (/MIX/i.test(transition) && transitionProps.frames !== undefined) { ... }
```
instead of just and index.

Reply:
This would be a very nice addidtion. Unfortunately `EFFEKT` doesn't have frames, instead it have the name of the breaker to run. So this suggestion, as in its current state, will break breakers